src/core/jwt/jwtTokenStore.js

```diff
@@ -55,6 +59,11 @@ const jwtTokenStoreFactory = function jwtTokenStoreFactory({
          * @returns {Promise<Boolean>} token successfully set
          */
         setAccessToken(token) {
+            if (usePerTokenTTL) {
+                const tokenPayload = parseJwtPayload(token);
+                const buffer = 10000;
+                currentAccessTokenTTL = (getJwtTTL(tokenPayload) || defaultAccessTokenTTL) - buffer;
```

Comment:
I think some config parameter have mixed purpose.
What does `accessTokenTTL` means exactly? Is it a default parameter, when ttl cannot be extracted from token? If yes, it should be described somehow in comments or with a better name.
As I see, this value lost its previous purpose and it doesn't have meaning if usePerTokenTTL is false, so I cannot go with a constant ttl. I think it is a breaking change.
If I can recommend something, I would go with a solution, where accessTokenTTL can be defined alone, without usePerTokenTTL.
If usePerTokenTTL and parse was successful, then usePerTokenTTL,
else if accessTokenTTL is defined then accessTokenTTL
else it won't expire at all. It won't cause issue, because when backend send 401, request module refreshed the token.

Reply:
My bad, I was not right.